### ISSUE TITLE
Show pipeline Duration time

### ISSUE THREAD

**Issue by mribichich** [karmats/gocd-monitor#10]
Hi there, first of all, great tool, it would be great to show the Duration of the pipeline, maybe next to when was build

if you want I can try to PR

thanks!

**Comment by karmats** [comment]
Hi @mribichich 

I'm not 100% sure what you mean by duration of the pipeline. Do you want to see how much time has passed when the pipeline started, i.e. just show this info when a pipeline is building?

**Comment by mribichich** [comment]
No, I mean the time the pipeline took to complete. How much time it was
spent on building it.
